registry/registry.go

```diff
@@ -99,7 +101,12 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d")
+			hostDir = filepath.Join(hostDir, req.URL.Host)
```

Comment:
might as well merge the two lines into `hostDir = filepath.Join(os.TempDir(), "/docker/certs.d", req.URL.Host)`